semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/span/SpanSemanticConventionUtils.java

```diff
@@ -144,4 +147,23 @@ public static String getPeerServiceName(Event event) {
     }
     return null;
   }
+
+  public static boolean isEmptyAttributesMap(Event event) {
+    return Optional.ofNullable(event.getAttributes()).map(Attributes::getAttributeMap).isEmpty();
+  }
+
+  public static boolean isValueNotNull(AttributeValue attributeValue) {
+    return Optional.ofNullable(attributeValue).map(AttributeValue::getValue).isPresent();
+  }
+
+  @Nullable
+  public static String getStringAttribute(Event event, String attributeKey) {
+    AttributeValue value = SpanAttributeUtils.getAttributeValue(event, attributeKey);
+    return value == null ? null : value.getValue();
+  }
+
+  public static boolean isHttpResponseCookie(String requestHeaderAttributeKey) {
```

Comment:
We should add this method in `HttpSemanticConventionUtils` and get rid of it from `EnrichedSpanUtils` and `SpanSemanticConventionUtils`